llvm/lib/CodeGen/AtomicExpandPass.cpp

```diff
@@ -1562,6 +1564,8 @@ bool AtomicExpandImpl::isIdempotentRMW(AtomicRMWInst *RMWI) {
   case AtomicRMWInst::Sub:
   case AtomicRMWInst::Or:
   case AtomicRMWInst::Xor:
+  case AtomicRMWInst::USubCond:
+  case AtomicRMWInst::USubSat:
```

Comment:
Missing tests for this (but this optimization is likely to be deleted soon anyway). Missing any new tests in test/Transforms/AtomicExpand

Reply:
If the optimization is likely to be deleted soon, then removing this part of the change seems like a better idea than adding new tests for it, so I have done that.